scala/scala.bzl

```diff
@@ -1164,10 +1164,11 @@ scala_junit_test = rule(
   attrs= _launcher_template + _implicit_deps + _common_attrs + _junit_resolve_deps + {
       "prefixes": attr.string_list(default=[]),
       "suffixes": attr.string_list(default=[]),
+      "suite": attr.label(default=Label("//src/java/io/bazel/rulesscala/test_discovery:test_discovery")),
```

Comment:
maybe `suite_label` and `suite_class` are more descriptive names (also in consultation with @ittaiz )

Reply:
Good idea. Ittai? WDYT?

Reply:
SGTM

Reply:
👍 done